docs/content/en/docs/getting-started/baremetal/bare-spec.md

```diff
@@ -306,7 +306,7 @@ In the example, there are `TinkerbellMachineConfig` sections for control plane (
 The following fields identify information needed to configure the nodes in each of those groups.
 >**_NOTE:_** Currently, you can only have one machine group for all machines in the control plane, although you can have multiple machine groups for the workers.
 >
-### hardwareSelector (optional)
+### hardwareSelector (Required)
```

Comment:
just to maintain consistency across other sections 
```suggestion
### hardwareSelector (required)
```

Reply:
done